doc/source/parameters/resolved_cfd-dem/resolved_cfd-dem.rst

```diff
@@ -138,6 +138,11 @@ The following properties are used if the particle impact one of the boundaries o
 
 * The ``lubrication range max`` parameter defines the distance below which the lubrication force between 2 particles or between a particle and a wall is calculated. The range is defined as a multiple of the smallest cell. The lubrication force model is used to model the force between particles when they are too close to each other to accurately resolve the flow between them.
 
+* The ``load particles from a file`` boolean that define if the particles are generated from a file instead of the parameter function. If this parameter is activated the particle defined in the particle's subsection are ignore as well as the number of particles. The number of particles is define directly from the file.
```

Comment:
```suggestion
* The ``load particles from a file`` boolean defines whether the particles are generated from a file instead of the parameter function. If this parameter is activated, the particles defined in the particle's subsection and the number of particles are ignored. The number of particles is defined directly from the file.
```